boa/test/plugin.py

```diff
@@ -15,8 +16,15 @@ def _HypothesisHandle__init__(self, *args, **kwargs):
 
     t = self.inner_test
 
+    has_profiling_flag = False
+    if "pytestmark" in t.__dict__:
+        for mark in t.pytestmark:
+            if mark.name == "profile_calls":
+                has_profiling_flag = True
+                break
+
     def f(*args, **kwargs):
-        with boa.env.anchor():
+        with boa.env.anchor(), boa.env.store_call_profile(has_profiling_flag):
```

Comment:
i don't think this is required to do on every test run, since the pytest machinery should already take care of it.

Reply:
it does not do so for hypothesis (which is why it is here).